### PR TITLE
Feat/sms: batches list

### PR DESCRIPTION
**Pull request by JPPortier**

### CODE REVIEW

File: openapi-contracts/src/main/com/sinch/sdk/domains/sms/adapters/api/v1/BatchesApi.java [hunk 1]
```diff
@@ -55,12 +55,11 @@ public BatchesApi(
...
-   * it&#39;s currently being delivered some messages currently being processed might still be
-   * delivered. The delivery report will indicate which messages were canceled and which
-   * weren&#39;t. Canceling a batch scheduled in the future will result in an empty delivery report
-   * while canceling an already sent batch would result in no change to the completed delivery
-   * report.
+   * Cancel a batch message A batch can be canceled aty point. If a batch is canceled while it&#39;s
+   * currently being delivered some messages currently being processed might still be delivered. The
+   * delivery report will indicate which messages were canceled and which weren&#39;t. Canceling a
+   * batch scheduled in the future will result in an empty delivery report while canceling an
+   * already sent batch would result in no change to the completed delivery report.
```
Comment (Dovchik) [inline openapi-contracts/src/main/com/sinch/sdk/domains/sms/adapters/api/v1/BatchesApi.java:62]: This paragraph has typos and sentence problems

**Review by Dovchik — APPROVED** [review]
LGTM but fix confusing paragraph

File: openapi-contracts/src/main/com/sinch/sdk/domains/sms/adapters/api/v1/BatchesApi.java (hunk 1 above)
Comment (JPPortier) [reply openapi-contracts/src/main/com/sinch/sdk/domains/sms/adapters/api/v1/BatchesApi.java:62]: Yes, the content is directly extracted from OAS file without any modification

**Review by 650elx — APPROVED** [review]
LGTM

**Pull request merged**